core/deployment/src/main/java/io/quarkus/deployment/steps/BannerProcessor.java

```diff
@@ -38,8 +42,12 @@ private String readBannerFile(BannerConfig config) {
         if (resource != null) {
             try (InputStream is = resource.openStream()) {
                 byte[] content = FileUtil.readFileContents(is);
-                String bannerTitle = new String(content, StandardCharsets.UTF_8);
-                return bannerTitle + "\n:: Quarkus :: v" + Version.getVersion() + '\n';
+                StringBuilder bannerTitle = new StringBuilder(new String(content, StandardCharsets.UTF_8));
+                bannerTitle.append('\n');
+                if (!config.isDefaultPath()) {
```

Comment:
this will only trigger if users override the banner name and don't just put `default_banner.txt` into their classpath root ...which I would think most would do because then they don't need to add anything to `application.properties`

should we instead have it be that if default path can't be resolved then we load our internal resource ?

Reply:
Currently what happens is the following:

If the user doesn't specify anything, the default is used. If the user does specify something that is not resolvable from the classpath root, we print a warning message.
If it is resolvable, we show it and also show the "Powered by..." text

Reply:
that’s not what I'm seeing:

if I have a file in my project called `src/main/resources/default_banner.txt` it uses my custom logo but `Powered by` is not added as `isDefaultPath()` is still true.

Reply:
Oh, now I see what you're saying. Yeah, I didn't think of that scenario and always changed the name. 
But if you do that, there is no way to load the internal `default_banner.txt`, right?

Reply:
Maybe we should just always show `Powered by Quarkus` and be done with it

Reply:
i have an idea - let me try code it up.

Reply:
Nevermind my question above. We definitely can load all `default_banner.txt`. I'll take a look

Reply:
PR to updated to take your use case into account